README.md

```diff
@@ -1,2 +1,3 @@
 # homepage
 My very own personal website. Basically this is just a résumé.
+Like and subscriiiiiibe))
```

Comment:
I would change to 'subscriiibe'

Reply:
Ok, would it be better to write with only one 'i'?